modules/apps/object/object-dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Attachment/Attachment.tsx

```diff
@@ -237,10 +243,11 @@ export default function Attachment({
 	};
 
 	return (
-		<FieldBase {...otherProps} {...error}>
+		<FieldBase {...otherProps} {...error} readOnly={readOnly}>
```

Comment:
Hey @Manda-vit, Just to let you know that we use a pattern here to always pass the rest props as the last of the component, like:

```tsx
<FieldBase readOnly={readOnly} {...otherProps} {...error}>
```